qaAuto.py

```diff
@@ -0,0 +1,65 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+
+def setup_driver():
```

Comment:
це треба викор стовувати як pytest fixture

```suggestion
@pytest.fixture
def setup_driver():
```